### PR TITLE
Update logo and link to marketing website

### PR DESCRIPTION
**Pull request by braden-holgate**
1. Update logo
2. Link logo to immersve.com
3. Add separate navbar item and page title "Docs"
4. Link title to '/'

Screenshot 2023-07-12 at 10 19 38 AM

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -50,4 +50,15 @@ footer {
...
   --ifm-color-primary-lightest: #bbd4fe;
-}
\ No newline at end of file
+}
+
+@media (min-width: 997px) {
+  .navbar-title {
+    position: absolute;
+    margin-top: 20px;
+    margin-left: 283px;
```
Comment (tonykha123) [inline src/css/custom.css:59]: if you change margin left to 270px, itll line up with the end of the sidebar. Personally I think it looks nicer lined up 
image

Comment (braden-holgate) [reply src/css/custom.css:59]: Hmmm. That doesn't do it for me. Any other reviewers have feelings on this?

Comment (catluin) [reply src/css/custom.css:59]: I also think it looks better with a bit of a gap. Looks a bit strange lined up with the border like this

Comment (tonykha123) [reply src/css/custom.css:59]: fair enough! personal preference

**Review by catluin — APPROVED** [review]

**Pull request merged**